build.gradle.kts

```diff
@@ -68,6 +71,11 @@ fun log4j(
 
 tasks.getByName("test", Test::class).apply {
     maxHeapSize = "2g" // Work around https://ecosystem.atlassian.net/browse/JPERF-395
+
+    testLogging {
+        events = setOf(TestLogEvent.PASSED, TestLogEvent.SKIPPED, TestLogEvent.FAILED)
```

Comment:
Gradle is quiet by design. If you want to access the stdout, open the test report HTML.
If you need it in real time, then you can run the test via IDEA.

Reply:
But with Travis it is currently close to impossible to see failure reason without build artefacts. This is an easy and not costly workaround.

Reply:
Ah, that makes sense. Let's include this rationale in the commit message. If we port to CircleCI, it would no longer apply.

Reply:
Done.